src/client/layouts/ConsentsLayout.tsx

```diff
@@ -117,7 +119,7 @@ export const ConsentsLayout: FunctionComponent<ConsentsLayoutProps> = ({
             </div>
             <ConsentsBlueBackground>
               <div css={[gridItem(gridItemColumnConsents), controls]}>
-                {!error && (
+                {!error && !hideContinueButton && (
```

Comment:
just a thought - maybe is this was called `showContinueButton` it might help with readability as we wouldn't have the cognitive load of the not operator

Reply:
Good shout, I went with this as by default we want it to show rather than hide. But I've refactored it after this suggestion to `showContinueButton` and setting the default value to true, so you have to explicitly pass `showContinueButton={false}` to hide it.

Reply:
nice :)